### PR TITLE
AF-2467 : Commit history in metrics does not get refreshed when using other than master branch

### PR DESCRIPTION
**Pull request by akumar074**

### CODE REVIEW

**Comment by kie-ci10** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

File: kie-wb-common-screens/kie-wb-common-contributors/kie-wb-common-contributors-backend/src/main/java/org/kie/workbench/common/screens/contributors/backend/dataset/ContributorsManager.java [hunk 1]
```diff
@@ -150,7 +151,10 @@ public DataSet buildDataSet(Map<String, String> params) {
                 for (final WorkspaceProject project : projects) {
                     final String repoAlias = project.getRepository().getAlias();
                     final String projectName = project.getName();
-                    org.uberfire.backend.vfs.Path rootPath = project.getRootPath();
+                    final Branch branchName = (params != null && params.get("activeBranch") != null) ?
```
Comment (adrielparedes) [inline kie-wb-common-screens/kie-wb-common-contributors/kie-wb-common-contributors-backend/src/main/java/org/kie/workbench/common/screens/contributors/backend/dataset/ContributorsManager.java:154]: @akumar074 thanks for the PR, I have a few comments here:
* "activeBranch" should be constant. 
* I think you should check if the branch exist before doing ".get()" or at least .orElse(project.getBranch());
* Tests are needed. Will not approve without them.

File: kie-wb-common-screens/kie-wb-common-contributors/kie-wb-common-contributors-backend/src/test/java/org/kie/workbench/screens/contributors/backend/dataset/ContributorsManagerTest.java [hunk 2]
```diff
@@ -595,4 +599,29 @@ public void testBuildDataSet() throws Exception {
         DataSetMetadata metadata = dataSet.getMetadata();
         assertNotNull(metadata);
     }
+
+    @Test
+    public void testGetBranch() {
```
Comment (adrielparedes) [inline kie-wb-common-screens/kie-wb-common-contributors/kie-wb-common-contributors-backend/src/test/java/org/kie/workbench/screens/contributors/backend/dataset/ContributorsManagerTest.java:604]: I think some cases are missing
* params.put("activeBranch", ""); case is missing (With this you will best the .orElse() statement)
* Also if it is null (Just to be sure everything works)

**Comment by adrielparedes** [comment]
@akumar074 why the build isn't woking? (I have a problem with my VPN and I can't get the result)

File: kie-wb-common-screens/kie-wb-common-contributors/kie-wb-common-contributors-backend/src/test/java/org/kie/workbench/screens/contributors/backend/dataset/ContributorsManagerTest.java (hunk 2 above)
Comment (akumar074) [reply kie-wb-common-screens/kie-wb-common-contributors/kie-wb-common-contributors-backend/src/test/java/org/kie/workbench/screens/contributors/backend/dataset/ContributorsManagerTest.java:604]: Hi @adrielparedes , thanks for reviewing the PR. Sorry, I have missed this. Thanks a lot for the suggestion. I'll update and re-request for review.

**Comment by akumar074** [comment]
> @akumar074 why the build isn't woking? (I have a problem with my VPN and I can't get the result)

@adrielparedes , There is some regression in JBPM repo due to these changes. I'm looking into it.

**Comment by akumar074** [comment]
jenkins retest this

**Comment by akumar074** [comment]
The implementation is changed so will raise a new PR.

**Pull request closed without merging**